src/hls-adapter.js

```diff
@@ -1169,22 +1169,28 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
 
   /**
    * Tries to handle media errors via hls.js error handlers
+   * @param {string} mediaErrorName - Media Error Name
    * @returns {boolean} - if media error is handled or not
    * @private
    */
-  _handleMediaError(): boolean {
+  _handleMediaError(mediaErrorName: string): boolean {
     const now: number = performance.now();
     let recover = true;
-    if (this._checkTimeDeltaHasPassed(now, this._recoverDecodingErrorDate, this._config.recoverDecodingErrorDelay)) {
+    if (mediaErrorName === Hlsjs.ErrorDetails.MANIFEST_INCOMPATIBLE_CODECS_ERROR) {
```

Comment:
is there an error number we can use or is there only a string to identify this error ?

Reply:
no seems hls js only expose MEDIA_ERROR and there we have the details
<img width="1286" alt="Screen Shot 2023-03-13 at 21 20 15" src="https://user-images.githubusercontent.com/16818804/224809820-eb4d03e0-94bb-4bdd-9058-d3b5935cf3a9.png">